.github/workflows/presubmit.yml

```diff
@@ -0,0 +1,51 @@
+name: PR Presubmit Reviews
+
+permissions:
+  contents: read
+  pull-requests: write
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled]
+  pull_request_review_comment:
+    types: [created]
+
+jobs:
+  review:
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'AI Review')
```

Comment:
Ok this should allow us to gatekeep this from running everywhere until we polish this